cmd/image/qcow2ova/prep/templates.go

```diff
@@ -37,8 +37,14 @@ rm -rf /etc/systemd/system/multi-user.target.wants/firewalld.service
 rpm -vih --nodeps https://public.dhe.ibm.com/software/server/POWER/Linux/yum/download/ibm-power-repo-latest.noarch.rpm
 sed -i 's/^more \/opt\/ibm\/lop\/notice/#more \/opt\/ibm\/lop\/notice/g' /opt/ibm/lop/configure
 echo 'y' | /opt/ibm/lop/configure
+{{if eq .Dist "rhel"}}
 # Disable the AT repository due to slowness in nature
 yum-config-manager --disable Advance_Toolchain
+{{end}}
+{{if eq .Dist "centos"}}
+yum-config-manager --add-repo=https://public.dhe.ibm.com/software/server/POWER/Linux/yum/IBM/CentOS/ppc64le/
```

Comment:
this centos repo is pretty old and we shouldn't using this, rather fallback to the rhel distro one.

Reply:
@mkumatag Pointed it to rhel repo